### PR TITLE
Button spacing

### PR DESCRIPTION
**Pull request by sarmand19**
added classes .leftformatButton and .actionMargin for status buttons in view-project-logic.components.css

### CODE REVIEW

**Review by BrandonSlade — APPROVED** [review]
The changes have been reviewed and I approve of them.

File: src/app/project/view-projects-logic/view-projects-logic.component.html [hunk 1]
```diff
@@ -33,13 +33,17 @@
 
       <!-- Status Column -->
       <ng-container matColumnDef="status">
-        <th mat-header-cell *matHeaderCellDef mat-sort-header> Status </th>
+        <!--<p> added by serge-->
```
Comment (mgoshorn) [inline src/app/project/view-projects-logic/view-projects-logic.component.html]: Is there any reason for this to stay in the source code?  If not, we should remove it and keep the code clean.

File: src/app/project/view-projects-logic/view-projects-logic.component.scss [hunk 2]
```diff
@@ -127,7 +127,18 @@
 }
 
 .formatButtons {
-  margin: 5px;
+ // margin: 5px;
```
Comment (mgoshorn) [inline src/app/project/view-projects-logic/view-projects-logic.component.scss]: Remove commented out configuration

File: src/app/project/view-projects-logic/view-projects-logic.component.scss [hunk 3]
```diff
@@ -127,7 +127,18 @@
...
+  margin: 10px;
+}
+
+//added by Serge
```
Comment (mgoshorn) [inline src/app/project/view-projects-logic/view-projects-logic.component.scss]: @sarmand19 Use an @Author decorator to mark your work.  No need to do this at the rule level though. Put it at the top of of the file for CSS.

**Review by mgoshorn — CHANGES_REQUESTED** [review]

**Review by mgoshorn — APPROVED** [review]

**Pull request merged**